package-lock.json

```diff
@@ -3626,11 +3626,6 @@
         "minimalistic-assert": "^1.0.0"
```

Comment:
please revert your change to package-lock.json. We will run an update to this ourselves once merged.

Reply:
Done